### DISCUSSION TITLE
can i get clean build please?

### DISCUSSION THREAD

**Discussion by Alone0316** [ROM-builders/temporary#13194] [General]
can i get clean build please? that is the only solution for that error as i have tried clean build 3 times and i am able to get it compiled perfectly without any problems

**Comment by Apon77** [comment]
No, clean build is not allowed

**Reply by Alone0316** [reply]
> No, clean build is not allowed

![Screenshot_20220911-194743_Plus](https://user-images.githubusercontent.com/70204635/189532643-c7ccb230-1479-470d-9592-b26c05d6400f.png)
![Screenshot_20220911-195238_Plus](https://user-images.githubusercontent.com/70204635/189532649-ed319890-1805-4c4d-9ae8-5b0145cab861.png)
![Screenshot_20220911-195244_Plus](https://user-images.githubusercontent.com/70204635/189532652-f4407d9d-1557-44fc-87d0-669c49180d67.png)
see i am able to get zip files, clean build will fix that, can you please allow a clean build? @Apon77
